content/about.html

```diff
@@ -0,0 +1,125 @@
+  </div>
+  <div class="about-modtext about-right" style="margin-top:10px">
+    <h3><img src="/images/about/mod-ra-logo.png" width="32" height="32" />Red Alert</h3>
+    <p>
+      In a world where Hitler was assassinated and the Third Reich never existed,
+      the Soviet Union seeks power over all of Europe. Allied against this Evil Empire,
+      the free world faces a Cold War turned hot.
+    </p>
+    <p>
+      The Red Alert mod focuses on <span class="about-highlight">strategy</span>, providing a range of
+      units and tactics to conquer the land, sea, and air.
+    </p>
+  </div>
+</div>
+
+<div class="about-modblock">
+  <div class="about-modimage about-right">
+    <img width="312" height="175" src="/images/about/ingame-d2k.png" />
+  </div>
+  <div class="about-modtext about-left" style="margin-top:5px">
+    <h3><img src="/images/about/mod-d2k-logo.png" width="32" height="32" />Dune 2000</h3>
+    <p>
+      Three great houses fight for the precious spice melange.<br />
+      He who controls the spice controls the universe!<br />
+      Establish a foothold on the desert planet Arrakis, where your biggest threat is the environment.
+    </p>
+    <p>
+      The Dune 2000 mod currently focuses on providing an experience that is
+      <span class="about-highlight">authentic</span> to the original game.
+    </p>
+  </div>
+</div>
+
+<div class="about-todo-divider"></div>
+
+<div class="about-modblock" style="height: 190px">
+  <div class="about-modimage about-left" style="margin-top: 4px">
+    <img width="312" height="175" src="/images/about/ingame-ts.png" />
+  </div>
+  <div class="about-modtext about-right">
+    <h3><img src="/images/about/mod-ts-logo.png" width="32" height="32" />Tiberian Sun</h3>
+    <p>
+      The next major goal for OpenRA is to add support for the second generation of
+      Command &amp; Conquer games, starting with Tiberian Sun.
+    </p>
+    <p>
+      The game engine features to support Tiberian Sun are still <span class="about-highlight">in development</span>.
+      The mod is not included with the OpenRA installer, but the source code is available in
+      <a href="https://github.com/OpenRA/OpenRA">our GitHub repository</a>.  Please note that we do not provide support
+      for incomplete development builds.
+    </p>
+  </div>
+</div>
+
+<div class="about-modblock">
+  <div class="about-modimage about-right">
+    <img width="312" height="175" src="/images/about/code.png" />
+  </div>
+  <div class="about-modtext about-left">
+    <h3><img src="/images/development.svg" width="32" height="32" />Custom Mods</h3>
+    <p>
+      We are working to cleanly separate the OpenRA engine code from our C&amp;C-specific mod details, so that it
+      can be used as a <span class="about-highlight">general purpose RTS game engine</span>.
+    </p>
+    <p>
+      It is already possible to build mods that extend our main OpenRA mods, or to build completely new mods with
```

Comment:
You may or may not want to link to  http://www.moddb.com/games/openra/mods here. On the one hand it would make sense, on the other, there is only one other link on that whole page.